fr/lander.yml

```diff
@@ -1,37 +1,137 @@
+  header: 
+    start_course: Commencer le cours
+  homepage:
+    learn_title: Apprenez à coder les DApps Blockchain en créant des jeux simples
+    learn_desc: CryptoZombies est une école de programmation interactive qui vous apprend à écrire des smart contracts en Solidity, en construisant votre propre jeu crypto.
+    why_different: Pourquoi CryptoZombies est différent
+    students: Étudiants
+    one_million: 1M+
+    classes_completed: Cours terminés
+    largest_ed_platform: CryptoZombies est la plus grande plate-forme éducative pour le développement de la blockchain, elle existe depuis plus de 4 ans, avec plus de 400 000 utilisateurs enregistrés qui ont terminé plusieurs cours.
+    actual_demand:  CryptoZombies a été le premier tutoriel sur Internet pour les NFT, et est toujours très pertinent pour la nouvelle génération de développeurs Web3 qui cherchent à entrer dans l'industrie aujourd'hui. Actuellement, le programme CryptoZombies est principalement axé sur le développement d'Ethereum et de Solidity, mais il y a beaucoup de demande pour du contenu ciblant d'autres chaînes telles que Binance, TRON et même des projets comme Chainlink.
+    interactive_coding_lessons: Interactive Coding Lessons
+    inbrowser_desc: Les leçons étape par étape dans le navigateur vous emmènent des bases mêmes de Solidity à la création de votre propre jeu entièrement fonctionnel basé sur la blockchain. Même à la fin de la leçon 1 (qui peut être complétée en une seule séance), vous en saurez assez pour vous appeler officiellement un développeur blockchain !
+    zombie_title: Construisez une armée de zombies
+    zombie_army: Après avoir terminé toutes les leçons et déployé votre DApp, opposez votre armée de zombies aux zombies des autres joueurs dans l'un des premiers jeux basés sur la blockchain au monde ! Moitié école de code, moitié jeu de stratégie crypto-collectible MMO.
+    collect_desc1: Développer le format du cours
+    collect_desc2: Nous allons étendre le format de cours existant à un format en direct et plus engageant hébergé dans le métaverse. Cela vous permettra de faire des événements de codage en direct, dans lesquels vous pourrez facilement partager votre code/écran avec un instructeur ou vos pairs, et interagir avec d'autres étudiants en utilisant le chat, l'audio ou la vidéo.
+    collect_desc3: Et vous connaissez le meilleur ? Tout est entièrement gratuit!
+    courses: Cours
+    roadmap: Feuille de route
+    roadmap_desc: La feuille de route illustre ce sur quoi nous travaillons. Nous le mettons à jour en permanence en fonction de vos commentaires.
+    migrate_telegram: Migrez la salle CZ Telegram vers notre chat communautaire
+    reputation_points: Points de réputation
+    participation_rewards: Points de réputation
+    participation_rewards: Récompenses de participation
```

Comment:
@mimiiiz This key is duplicated. I think "Récompenses de participation" is correct.

Reply:
thanks! removed